lib/magma/attribute.rb

```diff
@@ -49,7 +49,6 @@ def validation_object
     def json_template
       {
         name: @name,
-        model_name: self.is_a?(Magma::Link) ? link_model.model_name : nil,
```

Comment:
Does Timur not rely on this field? Sorry, I'm fairly ignorant of what fields Timur relies on, so just checking...

Reply:
I think you're right. I'll check Timur and make a corresponding PR if necessary before merging this one.

Reply:
We've generally been trying to make non-breaking changes in Magma so Timur will keep working. So if Timur depends on `model_name`, we should keep it here.

Reply:
We should add a replacement field (link_model_name) but keep this in place until we patch timur